org.jcryptool.visual.aco/src/org/jcryptool/visual/aco/view/AntColVisualComposite.java

```diff
@@ -1,19 +1,23 @@
 package org.jcryptool.visual.aco.view;
 
+//import org.eclipse.jface.resource.ImageDescriptor;
```

Comment:
For future reference: Commenting out unused imports is not necessary. Just let the Eclipse tools remove them altogether. It is in any case good practice to run "organize imports" on files or even whole projects that will be committed.